app/controllers/sufia/admin/admin_sets_controller.rb

```diff
@@ -115,5 +118,9 @@ def admin_set_params
       def repository_class
         blacklight_config.repository_class
       end
+
+      def permission_template_params
```

Comment:
This seems a bit hacky to me, but without separating the other admin_set params, the PermissionTemplate throws an error. If it's possible to indicate permission_template instead of admin_set on the form, this might be cleaner.